### ISSUE TITLE
[Accessibility][Pivot]: Control fails for button-name

### ISSUE THREAD

**Issue by satishc2437** [OfficeDev/office-ui-fabric-react#1074]
Hi,
We are seeing issues with Pivot control in our Keros run. Below are the details. Requesting you to please take a look.

 https://keros.cloudapp.net/kerosweb/#!/Tests?test=541cfd49-93ad-41af-9bc0-b4b8df1947d5&scenario=fac84221-7537-4665-bafc-6834c2917808

![image](https://cloud.githubusercontent.com/assets/13657916/23206089/57f3192a-f912-11e6-8336-517f4eb92efb.png)

Thanks,
Satish

**Comment by joschect** [comment]
@satishc2437 Could elaborate about what that error means? Does that mean that the button is missing a button-name attribute?

**Comment by battletoilet** [comment]
![image](https://user-images.githubusercontent.com/622498/35643294-b76eba5a-067a-11e8-8e86-d4137b6fb3ef.png)
